### PR TITLE
PMM-6827 DB cluster monitoring.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: api/serverpb/server.proto [hunk 1]
```diff
@@ -147,6 +149,9 @@ message ChangeSettingsRequest {
   bool enable_stt = 11;
   // Disable Security Threat Tool
   bool disable_stt = 12;
+  // PMM Server public address.
+  string pmm_public_address = 13;
```
Comment (askomorokhov) [inline api/serverpb/server.proto:153]: What about pmm external address? :)

**Review by askomorokhov — APPROVED** [review]

File: api/serverpb/server.proto [hunk 2]
```diff
@@ -122,6 +122,8 @@ message Settings {
   string platform_email = 10;
   // DBaaS enabled
   bool dbaas_enabled = 11;
+  // PMM Server public address.
+  string pmm_public_address = 13;
```
Comment (JiriCtvrtka) [inline api/serverpb/server.proto:126]: Number 12 skipped because of consistence with Change Settings?

**Review by percona-csalguero — APPROVED** [review]

**Review by AlekSi — COMMENTED** [review]
Why is that address and not URL? Please add example value to the comment

**Comment by BupycHuk** [comment]
@AlekSi operators expects address, the same we use in PMM_AGENT_SERVER_ADDRESS

**Comment by AlekSi** [comment]
I'm pretty sure they also need username and password for `pmm-agent setup`

**Comment by BupycHuk** [comment]
@AlekSi at the moment they use secrets for the password in xtradb and for the username and password in psmdb. Maybe we can leave it for the time when we will handle secrets creation? Just to not confuse users who will expect that if they set url with username and password it should work. WDYT?

**Comment by AlekSi** [comment]
Sure

File: api/serverpb/server.proto (hunk 1 above)
Comment (BupycHuk) [reply api/serverpb/server.proto:153]: isn't it the same? public sounds better for me :)

File: api/serverpb/server.proto (hunk 2 above)
Comment (BupycHuk) [reply api/serverpb/server.proto:126]: Will update before merge

File: api/serverpb/server.proto (hunk 1 above)
Comment (askomorokhov) [reply api/serverpb/server.proto:153]: This is not critical. :)
The word "public" has the connotation "public place". pmm-server not necessary to be publicly available for everybody.  It only must be available for external to its services.

**Pull request merged**